kinto_client/replication.py

```diff
@@ -0,0 +1,111 @@
+import argparse
+import logging
+
+from kinto_client import Client
+from kinto_client import exceptions
+
+logger = logging.getLogger(__name__)
+
+
+def replicate(origin, destination):
+    """Replicates all the information from one server to another one.
+
```

Comment:
You could maybe add a note to specify that it replicates *all records* and not just the one that changed.